src/actions/guides/database/models.cr

```diff
@@ -84,15 +175,13 @@ class Guides::Database::Models < GuideAction
         belongs_to post : Post
       end
     end
-
```

Comment:
I thin having a newline between the classes adds a nice separation. What do you think about adding them back?